tenable/io/v3/vm/permissions/schema.py

```diff
@@ -0,0 +1,21 @@
+'''
+Permissions API Endpoint Schemas
+'''
+from marshmallow import Schema, fields, validate
+
+
+class PermissionAclSchema(Schema):
+    '''
+    Schema for acls for change permissions action
+    '''
+    type = fields.Str(required=True, validate=validate.OneOf(
+        ['default', 'user', 'group']))
+    id = fields.Int(required=True)
+    permissions = fields.Int(required=True)
```

Comment:
@harshshah-tenable Have you verified if both of them are required params? The spec here doesn't mention it as required.

Reply:
Yes validated again these are not required params in payload for API removing the required keyword from schema